src/audio/drc/Kconfig

```diff
@@ -3,6 +3,7 @@
 config COMP_DRC
 	bool "Dynamic Range Compressor component"
 	select CORDIC_FIXED
+        select MATH_LUT_TRIG_FIXED
```

Comment:
Can it be `MATH_LUT_SINE_FIXED` instead of `MATH_LUT_TRIG_FIXED`?

Reply:
Yep, I'll change the config name. There is no need for other functions now.